shell/platform/darwin/ios/ios_external_texture_gl.h

```diff
@@ -12,34 +12,41 @@
 
 namespace flutter {
 
-class IOSExternalTextureGL : public flutter::Texture {
+class IOSExternalTextureGL final : public Texture {
```

Comment:
nit: for large PR's we should probably split out refactoring into a different PR.  No need to change anything here, just food for thought.